kasimx:
  # Publish all joint states -----------------------------------
  # joint_state_controller:
  #   type: joint_state_controller/JointStateController
  #   publish_rate: 50  

  # Velocity Controllers ---------------------------------------
  joint1_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: left_arm1
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint2_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: left_arm2
    pid: {p: 100.0, i: 0.01, d: 10.0}